### ISSUE TITLE
ui not refreshing

### ISSUE THREAD

**Issue by wtjerry** [mogria/rtsh#2]
(maybe also a feature)

it would be nice to have the ui refreshing after either a certain amount of time or after specific commands (like move, create..)

**Comment by mogria** [comment]
The idea was that the node js `fs.watch` looks at the /world directory and constantly checks for changes. If a change occurs only the file that changed is sent again to the client via websocket, so he can display the update instantly.

At the moment only the initial sending and displaying of the board works properly.

**Comment by mogria** [comment]
file changes are now propagated to the client, removals don't quite work yet. This will need some more javascript code. The client side javascript should be organized better. I think about using require.js or something similar.

**Comment by wtjerry** [comment]
check out branch unlinkIsBuggy on wetty-cli. it might fix this issue, but not sure if fully.

**Comment by mogria** [comment]
This just forces the whole map to be sent again. The `init-*` events are thought to occur only once, and this should be when reloading the page.

what needs to be done is to be able to send a message that a file has been deleted. So instead of the typical `{file: "filename."json", data: "{ some json content }"}`, we  need to send a message like `{file: "filename.json", removed: true}`. Then the client side javascript can check for this and call the appropriate functions.
